overlays/mingw_w64.nix

```diff
@@ -74,6 +81,7 @@ let
   wineTestWrapper = writeScriptBin "test-wrapper" ''
     #!${stdenv.shell}
     set -euo pipefail
+    export TMP=$(mktemp -d)
```

Comment:
NICE!

Reply:
Oh that is not needed now.  I tried it when debugging he wine issue that was fixed by updating tulia.

Reply:
Oh, lol! :D